src/api/exporters/ezsLocals.js

```diff
@@ -84,7 +85,26 @@ export function JSONLDObject(data, feed) {
                 }
             }
         });
-        feed.send(output);
+
+        const transformCompleteFields = async (field) => {
+            const propertyName0 = await generateUid();
+            const propertyName1 = field.name;
+            const propertyName2 = field.completes;
+            output[propertyName0] = {};
+            output[propertyName0][propertyName1] = data[propertyName1];
+            output[propertyName0][propertyName2] = data[propertyName2];
+            delete output[propertyName2];
+            output['@context'][propertyName0] = {};
+            output['@context'][propertyName0]['@id'] = output['@context'][propertyName2]['@id'];
+            output['@context'][propertyName2]['@id'] = 'http://www.w3.org/2000/01/rdf-schema#label';
+        };
+        const allFields = fields
+            .filter(field => field.completes !== undefined)
```

Comment:
FYI: You don't need the `!== undefined` here. When simply returning `field.completes` it will be falsy if `null` or `undefined`